test/global-compress.test.js

```diff
@@ -3048,3 +3048,71 @@ test('It should return an error when using `reply.compress()` with a missing pay
     statusCode: 500
   }, payload)
 })
+
+const defaultSupportedContentTypes = [
+  'application/json',
+  'application/json; charset=utf-8',
+  'application/graphql-response+json',
+  'application/graphql-response+json; charset=utf-8',
+  'application/xml',
+  'application/xml; charset=utf-8',
+  'octet-stream',
```

Comment:
I realize now, that this is wrong. Should be application/octet-stream. And application/octet-stream does not match?!

Reply:
Correction. application/octet-stream is also matching. Probably a bug.

Reply:
does #262 supersedes this PR?

Reply:
No. It should result in a merge conflict, which should be resolved.